buf.yaml

```diff
@@ -0,0 +1,25 @@
+version: v1beta1
+build:
+  roots:
+    - .
+  excludes:
+    - vendor
+lint:
+  use:
+    - BASIC
```

Comment:
Was there a reason you chose this list? The best to start with is `DEFAULT`, and go from there. Note that `buf check lint --error-format=config-ignore-yaml` will generate a lint ignores section you can copy/paste to ignore your current errors on a per-file level.

Reply:
This list is a combination of something I copied from the docs as well as the output of the command you gave.

Reply:
Ah - that list is just meant for the tour, it's used for googleapis as they have a lot of things that don't conform (semi-obviously). We recommend starting with:

```yaml
lint:
  use:
    - DEFAULT
```

And then going from there.

You can see all available categories and checkers with `buf check ls-lint-checkers --all` as well. The introduction to https://docs.buf.build/lint-checkers gives a rough overview of the hierarchy.